tox.ini

```diff
@@ -114,6 +114,7 @@ markers =
     timeout: tests that need a timeout time
     internal: tests that could but should only run on our internal CI
     external: tests that could but should only run on our external CI
+    backend: used in the sqlalchemy 2.0 test suite
```

Comment:
SQLAlchemy 2.0 test suite adds a marker and since `--strict-markers` is set, pytest will refuse to run the tests without it added in the config. Doesn't affect 1.4.